usage_record_summary.go

```diff
@@ -0,0 +1,25 @@
+package stripe
+
+// UsageRecordSummary represents a usage record summary.
+// See https://stripe.com/docs/api#usage_records
+type UsageRecordSummary struct {
+	ID               string  `json:"id"`
+	Live             bool    `json:"livemode"`
```

Comment:
I think we agreed on the fully expanded `Livemode` for these types of properties, but doh! I just noticed when grepping that we missed a couple places — usage records, issuer fraud records, and source transactions.

I'm not sure what to do with those ones right now, but can we make this one `Livemode`?

Reply:
Damn nice catch, I copy/pasted from usagerecord and when I double checked I saw other matches and did not look further